lib/util/viewer.js

```diff
@@ -156,3 +156,53 @@ export function stopTimeComparator (a, b) {
   const bTime = b.serviceDay + b.scheduledDeparture
   return aTime - bTime
 }
+
+/**
+  * Find the stop time corresponding to the first departure
+  * (the closest departure past the current time).
+  */
+export function getFirstDepartureFromNow (mergedStopTimes) {
+  // Search starting from the last stop time (largest seconds until departure) for this pattern.
```

Comment:
I think we should probably delete "for this pattern". Might be a vestige from previous code.

Reply:
Yeah I missed that one. Fixed in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/ced7d9221f0ad6ada60563f452610feb8f9ed297.